hw3es/hw3es/Vehicles.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+
+namespace hw3es
+{
+    public class Vehicle
+    {
+        public double CoordinateX { get; set; }
+        public double CoordinateY { get; set; }
+
+        public double PriceUsd { get; set; }
+        public float SpeedKms { get; set; }
+        public int Year { get; set; }
+
+        public Vehicle(double coordinateX, double coordinateY, double priceUsd, float speedKms, int year)
+        {
+            CoordinateX = coordinateX;
+            CoordinateY = coordinateY;
+            PriceUsd = priceUsd;
+            SpeedKms = speedKms;
+            Year = year;
+        }
+
+        public void Show()
```

Comment:
why have you decided not to make it `virtual`? should behavior of this method change when object of `Plane` for example is upcasted?